RUFAS/biophysical/manure/handler/single_stream_handler.py

```diff
@@ -0,0 +1,204 @@
+from typing import Any
+
+from RUFAS.biophysical.manure.handler.handler import Handler
+from RUFAS.current_day_conditions import CurrentDayConditions
+from RUFAS.data_structures.animal_to_manure_connection import ManureStream
+from RUFAS.biophysical.manure.manure_constants import ManureConstants
+from RUFAS.data_structures.processor_types import ProcessorTypes
+from RUFAS.time import Time
+from RUFAS.units import MeasurementUnits
+
+
+DEFAULT_PH_FOR_HOUSING_AMMONIA: float = 7.7
+"""Default pH for housing ammonia (unitless). Default is set to 7.7."""
+
+HOUSING_HSC = 260.0  # s/m
+"""
+Default housing specific constant (s/m). This constant may be used in calculations
+related to the housing conditions for animals. Default is set to 260.0 s/m.
+"""
+
+
+class SingleStreamHandler(Handler):
+    """
+    Base class for all handlers that only accepts single manure stream at a time.
+
+    Parameters
+    ----------
+    name : str
+        Unique identifier of the processor.
+    handler_type: ProcessorTypes
+        The class of sun type of manure handlers that this handler falls into.
+    cleaning_water_use_amount : float
+        Amount of cleaning water used per animal per day (L).
+    minutes_per_cleaning : int
+        Number of minutes needed per animal per cleaning, minutes.
+    cleanings_per_day : int
+        Number of cleanings per day.
+    cleaning_water_recycle_fraction : float
+        Fraction of cleaning water that is from recycled (not fresh) water sources.
```

Comment:
```suggestion
        Fraction of cleaning water that is from recycled (not fresh) water sources (unitless).
```
I think because there are units on a few of these that it might be helpful to add units on all of the numeric ones.